templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -1,6 +1,7 @@
 # ToDo Application with a C# API and Azure Cosmos DB SQL API
 
-[![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/todo-csharp-cosmos-sql)
+[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=446558288&machine=standardLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2)
```

Comment:
Silly that the link requires the unique identifier of the repo instead a reponame like `Azure-Samples/todo-nodejs-mongo`.

Reply:
Avoiding name lookups I guess